docs/acetate.config.js

```diff
@@ -125,15 +125,27 @@ module.exports = function(acetate) {
       (e, contents) => {
         const typedoc = JSON.parse(contents.toString());
 
-        const declarationPages = typedoc.declarations.map(declaration => {
-          return createPage.fromTemplate(
-            declaration.src,
-            path.join(acetate.sourceDir, "api", "_declaration.html"),
-            Object.assign({}, declaration, {
-              layout: "api/_layout:content"
-            })
-          );
-        });
+        // const declarationPages = typedoc.declarations.map(declaration => {
```

Comment:
let's remove this commented code and add this to rest-js too!

Reply:
I'll remove the commented code, but I want to give this more thought before we push this into rest-js. I would like to have the option to create "full" docs... what I've done here is pretty "harsh" fix... there may be another acetate hook we can use, or we can pre-parse the typedoc output... I will look at this now...